utils/shared.js

```diff
@@ -1039,6 +1039,149 @@ const updateBaselineData = (biospecimenData, participantData, siteTubesList) =>
     return participantUpdates
 }
 
+/**
+ * Broken into a separate helper function for easier testing
+ */
+const getHomeMWReplacementKitData = (data) => {
+    let fieldPath;
+    if(data?.[fieldMapping.collectionDetails]?.[fieldMapping.baseline]?.[fieldMapping.bioKitMouthwashBL2]) {
+        // If two replacements, they are out of replacement kits; error out.
+        throw new Error('Participant has exceeded supported number of replacement kits.');
+    }
+    
+   const participantIsEligible = !!processParticipantHomeMouthwashKitData(data, true);
+
+   if(!participantIsEligible) {
+    throw new Error('Participant address information is invalid.');
+   }
+
+   
+    if(data?.[fieldMapping.collectionDetails]?.[fieldMapping.baseline]?.[fieldMapping.bioKitMouthwashBL1]) {
```

Comment:
NIT - assigning some of these super long field paths to variables might help readibility